fastai/callback/mixup.py

```diff
@@ -31,7 +36,11 @@ def after_cancel_train(self):
     def after_cancel_fit(self):
         self.after_train()
 
-    def lf(self, pred, *yb):
+    def lf(self,
+        pred,
+        *yb
+    ):
+        "lf is a loss function"
```

Comment:
Can we be a bit more accurate in what this is? While it is a loss function, it's a bit more than that